addons/rose/addon/components/rose/form/fieldset/index.hbs

```diff
@@ -0,0 +1,20 @@
+<fieldset ...attributes class='rose-form-fieldset'>
+    <legend 
+        id={{concat 'title-' this.id}}
+        aria-describedby={{concat 'description-' this.id}}
+        class='rose-form-fieldset-title'
+    >
+        {{yield to='title'}}
```

Comment:
I like the use of named blocks.

Reply:
@gsusmi show me about them so I kindda cheat here :stuck_out_tongue_winking_eye: